Semver/Ranges/SemVersionExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Semver.Ranges.Comparers.Npm;
+
+namespace Semver.Ranges
+{
+    public static class SemVersionExtensions
```

Comment:
I think these should be methods on `SemVersion` so that there are no issues with them not being available because the namespace isn't imported.